slides/hypospaces-capacity/slides-hypospaces-complexity.tex

```diff
@@ -7,7 +7,7 @@
   \item Know PAC learning
   \item Know that there is no "universal" learner which works on every task (no free lunch)
   \item Know that complexity of a hypothesis space can be measured by VC dimension
-  \item Know that a hypothesis space is PAC learnable iff it has finite VC dimension
+  \item Know that a hypothesis space is PAC learnable if it has finite VC dimension
```

Comment:
das hier könnte auch so gemeint sein, "iff" heißt dann "if and only if" => wenn du dir nicht sicher bist oder es nachgeschaut hast würde ich es so lassen

Reply:
Hab nachgeschaut, "iff" ist richtig 👍  sorry my bad!